report/src/main/java/org/jboss/pnc/build/finder/report/HTMLReport.java

```diff
@@ -153,10 +151,7 @@ private static ContainerTag errorText(String text) {
         if (build.isPnc()) {
             return error ? errorText(name)
                     : a().withHref(
-                            pncUrl + "/pnc-web/#/projects/" + build.getBuildInfo().getExtra().get("external_project_id")
-                                    + "/build-configs/"
-                                    + build.getBuildInfo().getExtra().get("external_build_configuration_id")
-                                    + "/build-records/" + build.getBuildInfo().getExtra().get("external_build_id")
+                            pncUrl + "/pnc-web/#/builds/" + build.getBuildInfo().getExtra().get("external_build_id")
```

Comment:
@jbartece This URL is not expanding to the correct location the way the builds URL does. Also, if PNC supports it, it would be nice to have a separate link for each artifact. This is the way it is in Koji, but I am not sure if PNC supports it. If PNC doesn't support it, then it is OK to link each artifact to the same page as long as the URL is valid.